planning_launch/config/planning_diagnostics/planning_error_monitor.param.yaml

```diff
@@ -0,0 +1,7 @@
+/**:
+  ros__parameters:
+    # trajectory check
+    error_interval: 100.0 # error interval distance threshold [m]
+    error_curvature: 2.0 # error curvature threshold
+    error_sharp_angle: 1.5708 # error sharp angle threshold
```

Comment:
This parameter is different from https://github.com/autowarefoundation/autoware.universe/pull/792/files#diff-8c29921c3b8bf47c6713c3156f89465d7760620cdeedb797ec1f0e1152ccb017R6.

Reply:
My missing. I will change a parameter in tuniverse.